agent/src/iconfucius/cli/chat.py

```
@@ -1322,136 +1395,153 @@ def run_chat(persona_name: str, bot_name: str, verbose: bool = False,
+    mcp_session = None
 
-    # Enable readline for input history (up/down arrows) and line editing
     try:
-        import readline  # noqa: F401
-    except ImportError:
-        pass
+        mcp_task, mcp_uv_server = await start_mcp_server(port=mcp_port)
+    except SystemExit:
+        return
+    except Exception:
+        pass  # MCP not available — fall back to direct execute_tool
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)